api-samples/readingList/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Reading List API Demo",
+  "version": "1.0",
+  "manifest_version": 3,
+  "description": "Uses the chrome.readingList API to display, update and remove reading list entries.",
+  "background": {
+    "service_worker": "sw.js"
+  },
+  "permissions": ["readingList"],
+  "action": {}
```

Comment:
@oliverdunk Sorry forgot to ask, can we add a minimum_chrome_version?

Reply:
Done!